.gitignore

```diff
@@ -9,3 +9,4 @@ node_modules
 images/settings
 gitlens-*.vsix
 tsconfig*.tsbuildinfo
+test-results
```

Comment:
Shouldn't need this because I moved the test-results into our `out` folder

Reply:
Ooh you're right, perhaps I had it from before that change. Removing it now